lib/cli/src/frameworks/lit/js/Button.js

```diff
@@ -0,0 +1,77 @@
+import { LitElement, html, css } from 'lit';
+import { styleMap } from 'lit/directives/style-map.js';
+
+export class StorybookButton extends LitElement {
+  static get styles() {
+    return css`
```

Comment:
If possible we should reuse the existing `./button.css` that is shared between EVERY CLI framework template. So it is easier for us to maintain.